lib/internal/per_context/primordials.js

```diff
@@ -175,5 +175,18 @@ primordials.SafePromise = makeSafe(
   copyPrototype(original.prototype, primordials, `${name}Prototype`);
 });
 
+// Create copies of abstract intrinsic objects that are not directly exposed
+// on the global object.
+// Refs: https://tc39.es/ecma262/#sec-%typedarray%-intrinsic-object
+[
+  { name: 'TypedArray', original: Reflect.getPrototypeOf(Uint8Array) },
+].forEach(({ name, original }) => {
```

Comment:
nit: we could make look more like the other blocks
```suggestion
  { name: 'TypedArray', from: Uint8Array },
].forEach(({ name, from }) => {
  const original = Reflect.getPrototypeOf(from);
```

Reply:
Having `original` in the entry object makes it possible to add additional primordials to this without depending on getting them from a subclass.